chirp/drivers/ft4.py

```diff
@@ -616,6 +616,9 @@ def get_memory_mappings(self, memory):
     (136000000, 174000000),    # VHF
     ]
 
+# Presumably the same for both models, this can be added to FT65E once tested
+VALID_OFFSETS_FT65 = (50_000, 99_950_000)
```

Comment:
I'm not sure what the units or meaning of this is really.. 50kHz to 99.950MHz?

Reply:
Yeah, apologies - this is just a range. The minimum configurable offset when programming via keypad is displayed as `0.050M` and you can increment in steps of 0.05 up to `99.950M`